package.json

```diff
@@ -60,5 +60,6 @@
     },
     "workspaces": [
         "packages/*"
-    ]
+    ],
+    "packageManager": "yarn@4.6.0"
```

Comment:
> When I run commands in the repo they are still run using yarn v1. Did you forget to commit some config?

This should be enough with the other comment. I don't have anything left locally

Reply:
And the CI does indeed use yarn v4